keyboards/quarkeys/z60/hotswap/Makefile

```diff
@@ -1,3 +1 @@
-ifndef MAKEFILE_INCLUDED
-	include ../../Makefile
-endif
+include ../../Makefile
```

Comment:
> This file should be removed.

The **entire** file needs to be deleted. Not just the line I commented on.

Reply:
ok, done